app/models/user.rb

```diff
@@ -33,4 +34,9 @@ def get_coords
         self.lon = geocode[:results].first[:lon]
       end
     end
+
+    def set_profile_picture
+      image = ImageService.new.user_image
+      self.profile_picture = image[:data][:attributes][:raw_image]
```

Comment:
See other comment about the fixture being just the `image_url` key while the model callback expects it to be json-api standards. 

Not a big deal and I'm assuming it's intentional, or just left behind from initial use of webmock instead of VCR.